sass/layout/_markdown.scss

```diff
@@ -196,48 +186,27 @@ h6 {
         }
     }
 
-    .hljs {
-        position: relative;
-        visibility: visible;
-        white-space: pre;
-        display: flex;
-        flex-direction: column;
-
-        >div {
-            display: flex;
-        }
-              
-    /* for block of numbers */
-    .hljs-ln-numbers:before {
-        content: counter(code-counter);
-        counter-increment: code-counter;
-        color: #ccc;
-        border-right: 1px solid #CCC;
-        vertical-align: top;
-        padding-right: 5px;
-        vertical-align: top;
-        padding-right: 4px;
+    .post-code__line-numbers {
+        @include user-select(none);
+        border-right: 1px solid #CCCCCC;
+        color: #CCCCCC;
```

Comment:
should any of the theme's variable color be considered for this one? it might not be visible on some themes otherwise

Reply:
These colours are a bit different since the background comes from the user's code theme which has 4 presets and no colour options. The two dark themes are visible in the screenshots above, but here's the light themes:
![Screen Shot 2020-08-24 at 9 25 47 AM](https://user-images.githubusercontent.com/3277310/91049877-df9e7300-e5eb-11ea-87a7-2715e3aee86c.png)
![Screen Shot 2020-08-24 at 9 25 59 AM](https://user-images.githubusercontent.com/3277310/91049878-e0370980-e5eb-11ea-8640-327c61cec612.png)
It might be a bit light, but I'd prefer to address that separately since we can't just do it based on the user's theme.